.github/workflows/weekly-acceptance-0-49-x.yml

```diff
@@ -0,0 +1,28 @@
+# Dispatch to the consul-k8s-workflows with a weekly cron
```

Comment:
Curt answered this question during his great presentation in our weekly leveling up meeting. 

It's needed in different files because of the Cron, we should still add that context in a comment :)